hbw/app/javascript/packs/hbw/components/shared/hoc/conditions.js

```diff
@@ -53,12 +53,20 @@ export default WrappedComponent => class WithConditions extends Component {
 
   getValue = (name) => {
     if (this.props.params.dynamic === true) {
-      return this.props.formValues[name];
+      let value = this.props.formValues[name];
+
+      if (typeof value === 'string') {
+        value = this.escapeDoubleQuotesAndBackslashes(value);
+      }
+
+      return value;
     } else {
       return this.variableByName(name);
     }
```

Comment:
It looks like we should escape values for static condition as well because it uses the same code for evaluation.